.config/zsh/010.env.zsh##o.Linux,e.zsh

```diff
@@ -0,0 +1,9 @@
+export PROMPT_REGISTRY_PROJECT_DIR="$HOME/.config/promptregistry"
+export REPO_PATH="$HOME/Development/Repos"
+
+# Linux-specific paths (no brew dependencies)
```

Comment:
This is almost the same than the Mac version. Please do it the other way around: one general file and one specific file for the brew stuff only for Mac